### DISCUSSION TITLE
Opinion about "Communicate using Markdown" course

### DISCUSSION THREAD

**Discussion by leonardo-collazo** [skills/.github#17] [General]
Is a very simple course

**Comment by gavdallas** [comment]
Hello please can you help me out as I have a problem with the 
![markdown](https://user-images.githubusercontent.com/110396852/182365983-029a0949-f08f-4920-9153-858fbd96328f.jpg)
Markdown course

**Reply by leonardo-collazo** [reply]
I just passed the Markdown course yesterday and didn't have any problems. Go to Step 1 and carefully check if you missed doing any task. If you doesn't resolve your issue, you can open a problem on the Issues tab.
